onnx/onnx.proto

```diff
@@ -37,11 +37,12 @@ package onnx;
 // - Attribute: These names identify attribute names for extra attributes that
 //   are passed to operators.
 // - Operator: These names identify particular operators.
-// - Tensor: These names identify intermediate tensor values flowing through
+// - Value: These names identify intermediate values (typically tensors) flowing through
```

Comment:
Whether or not we actually rename Tensor to Value is, I think, blocked on figuring out what happens in https://github.com/onnx/onnx/pull/67